Puppetfile

```diff
@@ -7,6 +7,7 @@ mod 'theforeman/dns',                :git => 'https://github.com/theforeman/pupp
 mod 'theforeman/git',                :git => 'https://github.com/theforeman/puppet-git'
 mod 'theforeman/puppetserver_foreman', :git => 'https://github.com/theforeman/puppet-puppetserver_foreman'
 mod 'theforeman/tftp',               :git => 'https://github.com/theforeman/puppet-tftp'
+mod 'puppet/redis',                  '>= 8.5.0'
```

Comment:
FTR, you might want to add this in the section with explanation headers (like why we pin postgresql and apache)